donation-web/src/main/java/com/donation/web/DataController.java

```diff
@@ -0,0 +1,216 @@
+package com.donation.web;
+
+import com.donation.crud.*;
+import com.donation.donor.*;
+import com.donation.donor.model.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/api/populateDatabase")
```

Comment:
Static data... it's ok to use it at the beginning but we will need another mechanism later